.github/workflows/test.yml

```diff
@@ -42,6 +42,9 @@ jobs:
           mix local.rebar --force
           mix deps.get
 
+      - name: Security Check
+        run: mix sobelow --private --skip --exit
+
       - name: Run Tests
         env:
           MIX_ENV: test
```

Comment:
Another optimization we should adopt as a pattern is to hoist this `MIX_ENV=test` up to the top of the whole workflow. If you run different steps with different `MIX_ENV` settings, you can end up compiling the whole app more than once, which makes every workflow run take more time. You can see that if you expand the `Security Check` and the `Run Tests` sections of this run: https://github.com/revelrylabs/slax/actions/runs/9180952793/job/25246558083